deployments/liqo/values.yaml

```diff
@@ -98,6 +98,9 @@ discovery:
   config:
     # -- Set a mnemonic name for your cluster
     clusterName: ""
+    # -- A set of labels which characterizes the local cluster when exposed remotely as a virtual node.
+    # It is suggested to specify the distinguishing characteristics that may be used to decide whether to offload pods on this cluster.
+    clusterLabels: {} # { "topology.kubernetes.io/zone": us-east-1 , "liqo.io/provider": your-provider }
```

Comment:
For the sake of clarity, I would move the examples on a new line below, one label per line

Reply:
yes, sure, thank you.